src/whitelist.js

```diff
@@ -45,17 +47,22 @@ const createEthContract = (web3, wallet, opt) => {
 		from: wallet.address
 	});
 	contract.defaultAccount = wallet.address;
+	contract.transactionConfirmationBlocks = 2;
```

Comment:
Is it a good idea to reduce the default number of confirmation blocks? Is it safe to assume the block gets inserted in the chain after 2?